### ISSUE TITLE
will be spasm supported-developed or archived?

### ISSUE THREAD

**Issue by 221V** [skoppe/spasm#59]
hello)

will be **spasm** supported-developed or archived?

just interesting, cause begin to work with D,
and interesed in D + WASM

but latest stable ldc2 is **1.39.0** now
but spasm for **1.20.0**
...
?

thank you :)

**Comment by etcimon** [comment]
I continued development and it has its own druntime, but there's a few regressions in LDC so you have to compile with LDC 1.36.0

Make sure you read the BUILDING.md file

https://code.dlang.org/packages/libwasm

**Comment by etcimon** [comment]
Also, this is a good example to get started, it features typescript support

https://github.com/etcimon/libwasm/tree/master/examples/dom-ts

**Comment by skoppe** [comment]
Hi @221V,

Thanks for taking an interesting in WASM and D.

> will be spasm supported-developed or archived?

Spasm started as an experiment for D and WASM but also to confirm that I could eliminate the Virtual DOM by essentially diffing at compile time and do as little HTML updates as possible. I consider it successful on both accounts.

There are a lot more things I wanted to try out and I had plenty of ideas to pursue, but the next chapter for me was porting DRuntime so that I could use the full D language in WASM. On that front I got pretty far (running druntime and phobos unittests in WASM engines), however, the GC didn't fully work and was causing issues. I have some ideas to solve this nowadays, but hardly any free time.

So long story short, I don't have any immediate plans for spasm and no time to dedicate to it either.

> but latest stable ldc2 is 1.39.0 now
> but spasm for 1.20.0

I don't see how newer versions of LDC won't support it. There might be some errors, but should be fairly small.
